source/vibe/http/server.d

```diff
@@ -690,6 +698,13 @@ final class HTTPServerRequest : HTTPRequest {
 			Remarks: Requires the HTTPServerOption.parseCookies option.
 		*/
 		Session session;
+
+		/** The settings of the server serving this request.
+		 */
+		@property const(HTTPServerSettings) serverSettings() const
+		{
+			return m_settings;
+		}
```

Comment:
I'd leave this as `package` for now to avoid any potential necessity for API breakage later. If it turns out that this is useful for external libraries/applications, we can still make it public.